hugr-py/src/hugr/ops.py

```diff
@@ -0,0 +1,135 @@
+from __future__ import annotations
+
+from abc import ABC, abstractmethod
+from dataclasses import dataclass, field
+from typing import Generic, TypeVar, TYPE_CHECKING
+from hugr.serialization.ops import BaseOp, OpType as SerialOp
+import hugr.serialization.ops as sops
+import hugr.serialization.tys as tys
+
+if TYPE_CHECKING:
+    from hugr._hugr import Hugr, Node
+
+
+class Op(ABC):
```

Comment:
Is there no way to unify this with the protocol in `_hugr.py`?
Can't we define the protocol here, and include it from there?

Reply:
accidental file include, gone now